src/staticanalysis/bot/default.nix

```diff
@@ -141,6 +143,10 @@ let
 
       # Expose gecko env in final output
       ln -s ${gecko-env}/bin/gecko-env $out/bin
+
+      # Use UTC as timezone
+      ln -s ${tzdata}/share/zoneinfo/UTC $out/etc/localtime
```

Comment:
It looks like this doesn't work as expeced, as we see `Uncaught Internal Error: (Sys_error "/usr/share/zoneinfo/: No such file or directory")`.

Reply:
It did work, this is a new issue, the original one has disappeared